.travis.yml

```diff
@@ -0,0 +1,14 @@
+# Documentation: http://docs.travis-ci.com/user/languages/julia/
+language: julia
+os:
+  - linux
+  - osx
+julia:
+  - release
```

Comment:
Don't currently include release as this is 0.3 which your code will never run on. Comment this out until 0.4 is released